### DISCUSSION TITLE
Is it possible to access part1 during installation?

### DISCUSSION THREAD

**Discussion by mendaxcat** [ventoy/Ventoy#1226] [Q&A] [answered]
Hello ventoy developers. Is it possible to access the part1 described in [ventoy doc](https://www.ventoy.net/en/doc_disk_layout.html) **during the installation of an OS**? I got an "device or resource busy" after trying to mount the partition 1 in debian 11.

It would be helpful to load some files when installing OS, for example some firmware. I am aware of creating another partition to hold the file, but it is not flexible and the partition size may not be desirable in the future. Thus, I want to put all files in one partition and that is the problem.

Anyway, thanks for your help in advance.

**Comment by ventoy** [comment, marked as answer]
The latest Ventoy 1.0.64  provides an experimental option  `VTOY_LINUX_REMOUNT` which can help to access the ISO partition after boot.
